esp32/README.md

```diff
@@ -91,6 +91,11 @@ $ git submodule init lib/berkeley-db-1.xx
 $ git submodule update
 ```
 
+You will also need to have PySerial installed if it isn't already.
+```bash
+$ pip install pyserial
+```
+
```

Comment:
Yeah, exactly, 'esptool' is a submodule so it's there in the file `$ESPIDF/components/esptool_py/esptool/esptool.py`, and can just get run from there.  Normally, though, you'd "install" it by running the setup.py, which would go get the dependencies as well.  Because we're skipping that step though ...